src/perform_step/low_order.jl

```diff
@@ -230,3 +230,50 @@ end
 
   end
 end
+
+#methods from https://doi.org/10.1016/j.cam.2009.11.010
+@muladd function perform_step!(integrator,cache::WangLi3SMil_AConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  #stage 1
+  k  = integrator.f(uprev,p,t)
+  u  = uprev + dt*k
+
+  #stage 2
+  k  = integrator.g(u,p,t)
```

Comment:
dividing by sqdt means that the effective diffusion distance is like dt, which means you want this evaluated at t, and so this should be `g(uprev,p,t)`

Reply:
![image](https://user-images.githubusercontent.com/26359940/58528466-30aa9600-81f4-11e9-95b5-c1d3111cb11e.png)
Basically this is the method.I don't want it evaluated at uprev.

Reply:
You're not exactly implementing that method though. You're doing a finite difference approximation to the g'. That's fine, but what I'm mentioning is that you need `k  = integrator.g(uprev,p,t)` in order to do that finite difference approximation between two `g` calculations that are `dt` spaced apart.

Reply:
I did that and the order was still approx 0.5.

Reply:
[Weaved test results 1](https://htmlpreview.github.io/?https://github.com/deeepeshthakur/Urls/blob/master/12.html)
[Weaved test results 2](https://htmlpreview.github.io/?https://github.com/deeepeshthakur/Urls/blob/master/13.html)

Reply:
Can you push the code for that?

Reply:
Oh this issue was resolved. Sorry I should have mentioned that.

Reply:
I still see it in the code?